ffwd-http-reporter/pom.xml

```diff
@@ -25,7 +25,7 @@
     <dependency>
       <groupId>com.spotify.ffwd</groupId>
       <artifactId>ffwd-http-client</artifactId>
-      <version>0.4.6</version>
+      <version>0.4.7</version>
```

Comment:
will it work with v0.4.7? https://github.com/spotify/ffwd-http-client/releases

Reply:
yeah im just having trouble with the mvn release timing out. once it successfully runs on my laptop we should be good to go